### PR TITLE
Update icons.json

### PR DESCRIPTION
**Pull request by 12th-devs**
Added several icon descriptions

### CODE REVIEW

File: icons.json [hunk 1]
```diff
@@ -1029,323 +1029,323 @@
...
   },
   {
     "emoji": "🐹",
-    "description": ""
+    "description": "hamster"
   },
   {
     "emoji": "🐺",
-    "description": ""
+    "description": rat""
```
Comment (StudioMovieGirl) [inline icons.json]: this

File: icons.json [hunk 2]
```diff
@@ -1029,323 +1029,323 @@
...
   },
   {
     "emoji": "👋",
-    "description": ""
+    "description": "wave, hello"
   },
   {
     "emoji": "👌",
-    "description": ""
+    "description": "ok, OK"
```
Comment (StudioMovieGirl) [inline icons.json]: capitalization does not matter

File: icons.json [hunk 3]
```diff
@@ -73,7 +73,7 @@
   },
   {
     "emoji": "⛽",
-    "description": "gas"
+    "description": "gas, fual, fual pump"
```
Comment (StudioMovieGirl) [inline icons.json]: fuel misspelled

File: icons.json [hunk 4]
```diff
@@ -1029,323 +1029,323 @@
...
   },
   {
     "emoji": "🐬",
-    "description": ""
+    "description": "dolphin"
   },
   {
     "emoji": "🐭",
-    "description": ""
+    "description": "moise"
```
Comment (StudioMovieGirl) [inline icons.json]: incorrect spelling

**Comment by 12th-devs** [comment]
So sorry, I am updating the PR right now

**Comment by StudioMovieGirl** [comment]
> So sorry, I am updating the PR right now

np thanks for the pr!

File: icons.json [hunk 5]
```diff
@@ -121,11 +121,11 @@
   },
   {
     "emoji": "🆎",
-    "description": ""
+    "description": "AB, ab"
```
Comment (StudioMovieGirl) [inline icons.json:124]: since capitalization doesn't matter we don't need both.

File: icons.json [hunk 6]
```diff
@@ -121,11 +121,11 @@
...
   },
   {
     "emoji": "🆑",
-    "description": ""
+    "description": "CL, cl"
```
Comment (StudioMovieGirl) [inline icons.json:128]: since capitalization doesn't matter we don't need both.

File: icons.json [hunk 7]
```diff
@@ -1029,323 +1029,323 @@
...
   },
   {
     "emoji": "🐄",
-    "description": ""
+    "description": "cow"
   },
   {
     "emoji": "🐅",
-    "description": ""
+    "description": "Cheetah"
```
Comment (StudioMovieGirl) [inline icons.json:1072]: I think this is a tiger.

**Pull request merged**